spk/synokernel-linuxtv/Makefile

```diff
@@ -0,0 +1,57 @@
+SPK_NAME = synokernel-linuxtv
+
+# Set package version based on kernel
+# kernel < 4.4 is frozon at version f6e3542
+# and too early in the build process to use
```

Comment:
You can include `spksrc.common.mk` here to access `version_lt`
I use this in spk/adminer/Makefile of #4514

Reply:
It actually is still too early to call it as TC_KERNEL is not yet defined neither.